src/drivers/uavcan_v1/parameters.c

```diff
@@ -77,32 +77,33 @@ PARAM_DEFINE_INT32(UAVCAN_V1_BAUD, 1000000);
  */
 PARAM_DEFINE_INT32(UAVCAN_V1_BAT_MD, 0);
 
+
 /**
  * UAVCAN v1 battery port ID.
  *
  * @min 1
- * @max 32767
+ * @max 65535
  * @group UAVCAN v1
  */
-PARAM_DEFINE_INT32(UAVCAN_V1_BAT_ID, 4242);
+PARAM_DEFINE_INT32(UAVCAN_V1_BAT_ID, 65535);
 
 // Subscription Port IDs
-PARAM_DEFINE_INT32(UCAN1_ESC0_PID, 0);
-PARAM_DEFINE_INT32(UCAN1_GPS0_PID, 0);
-PARAM_DEFINE_INT32(UCAN1_GPS1_PID, 0);
-//PARAM_DEFINE_INT32(UCAN1_BMS0_PID, 0);
-//PARAM_DEFINE_INT32(UCAN1_BMS1_PID, 0);
+PARAM_DEFINE_INT32(UCAN1_ESC0_PID, 65535);
```

Comment:
Why initialize to 65535 instead of 0?

Maybe quickly fill in the some of the parameter metadata for each to explain the magic value? It would also be good to get these in the same group for editing in QGC.
https://github.com/PX4/PX4-Autopilot/blob/0ff56ffc8328baa678ea088bab60f79d00e761cb/src/drivers/uavcan_v1/parameters.c#L46-L55

Reply:
65535 means unset which is defined here https://github.com/PX4/PX4-Autopilot/blob/f9d8c613b048f58eb3110e9af13cb3a89c4c866f/src/drivers/uavcan_v1/Publishers/Publisher.hpp#L62

Reply:
Can you specify a max value with an exception for 65535.
e.g. 
```
@min 0
@max 6143
@unset 65535
```

Is there a syntax definition for this?

Reply:
No, I think you'd need to change the metadata handling (which you can if you want).

Other options
 - add separate boolean parameters for each of these to enable/disable
 - in the module constrain the parameter from 0 to 6143, and any values outside of that treat as disabled and reset/commit to 65535
 - combine that with using 0 to disable?

Reply:
Yes that sounds good.